src/assets/i18n/en.json5

```diff
@@ -4937,6 +4941,12 @@
 
   "submission.sections.submit.progressbar.describe.steptwo": "Describe",
 
+  "submission.sections.submit.progressbar.describe.dataciterequired": "Required Properties",
```

Comment:
should it be in cs too?

Reply:
translated in [906657d](https://github.com/dataquest-dev/dspace-angular/pull/622/commits/906657dc72f07ce6fc3e423c0fbe63b58b8a26ec)